app/src/pages/inside/logsPage/modals/attachmentModal/attachmentModal.stories.jsx

```diff
@@ -21,15 +21,11 @@
 
 import { storiesOf } from '@storybook/react';
 import { withReadme } from 'storybook-readme';
-import {
-  AttachmentCodeModal,
-  AttachmentHarFileModal,
-  AttachmentImageModal,
-} from '.';
+import { AttachmentCodeModal, AttachmentHarFileModal, AttachmentImageModal } from '.';
 import README from './README.md';
 import harData from './example.har';
 
-storiesOf('Pages/Inside/LogsPage/AttachmentModal', module)
+storiesOf('Pages/Inside/LogsPage/AttachmentCodeModal', module)
```

Comment:
This should remain `AttachmentModal` as codeModal and harModal are part of it. Beside that it seems all good! Thanks

Reply:
Yes, linter said that Pages/Inside/LogsPage/AttachmentModal is not exists. I thought that you removed/renamed/refactored them. 
So now only attachmentCodeModal.jsx	, attachmentHarFileModal.jsx attachmentImageModal.jsx is available.

Reply:
@ypetya  please review